modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -366,6 +366,9 @@ private CompletableFuture<?> onUpdateReplicas(ConfigurationNotificationEvent<Int
             if (replicasCtx.oldValue() != null && replicasCtx.oldValue() > 0) {
                 TableConfiguration tblCfg = replicasCtx.config(TableConfiguration.class);
 
+                LOG.info("Received update for replicas number for table={} from replicas={} to replicas={}",
```

Comment:
If we run sone rebalance test, we can see in the logs the following:

```
2022-06-23 13:18:27:798 +0300 [INFO][Thread-242][TableManager] Received update for replicas number for table=PUBLIC.TBL1 from replicas=1 to replicas=2
2022-06-23 13:18:27:799 +0300 [INFO][Thread-244][TableManager] Received update for replicas number for table=PUBLIC.TBL1 from replicas=1 to replicas=2
2022-06-23 13:18:27:802 +0300 [INFO][Thread-241][TableManager] Received update for replicas number for table=PUBLIC.TBL1 from replicas=1 to replicas=2
```
Lines looks the same and could be more informative, if we add info about the node where such update has happened

Reply:
Maybe we should think about a general solution, where all lines of logs will contain info about the node which logged that line

Reply:
Yes, I thought about it. I think it must be the general solution, because this point connected with all current logs.